### PR TITLE
feature(Trysilplug): using custom labels as images

### PR DESCRIPTION
**Pull request by BazanAlexandro**
Added small image (icon) at the bottom of trysilplug image. 
Is used for video 'show' functionality
Design: https://app.zeplin.io/project/5be3f8e9e6db113f290d3fd5/screen/5c9cd6a41f295f0642264753
Using the same work flow taking OverlayingPlugLabel as an example

#### Please tick a box ###
- [x] **feature** _(A new feature_)
- [ ] **fix** _(A bug fix_)
- [ ] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [x] Yes
- [ ] No
- [ ] Not adding a new feature

#### Are the components you're working on reusable between brands?
- [x] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_
- [ ] Not working on any components

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/molecules/TrysilPlug.js [hunk 1]
```diff
@@ -87,20 +103,23 @@ const TrysilPlug = ({
...
-						src={placeholderUrl ? placeholderUrl : image}
-						preventBlur={preventBlur}
-						fadeIn={fadeIn}
-					>
-						{sources.length === 0
-						&& <Source srcSet={image} />
-						}
-						{sources.map((source, i) => <Source srcSet={source.url} media={source.media} key={`source-${i}`} />)}
-					</LazyProgressiveImage>
+					<Relative>
```
Comment (misund) [inline src/molecules/TrysilPlug.js]: We will not wrap our figure in a div.

Comment (BazanAlexandro) [reply src/molecules/TrysilPlug.js]: No clear, why not?
Ok, for such case I can use existing functionality, agree. But why not wrapping component in `div` for styling purposes?

Comment (BazanAlexandro) [reply src/molecules/TrysilPlug.js]: By the way, thanks, I haven't checked custom labels functionality. Works fine for me!

**Comment by BazanAlexandro** [comment]
By the way, do we still need to add our components to babel-transform-shiny.js?
Looks like it's not maintained right now...

File: src/atoms/OverlayingImageLabel.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import { Image } from './Image';
+
+const OverlayingImageLabel = styled(Image)`
```
Comment (evoX0) [inline src/atoms/OverlayingImageLabel.js]:
```suggestion
const OverlayingImage = styled(Image)`
```
how do you think

Comment (misund) [reply src/atoms/OverlayingImageLabel.js]: I like Alex's naming better, it tells me that this is a label that happens to be an image

**Comment by misund** [comment]
> do we still need to add our components to babel-transform-shiny.js?
> Looks like it's not maintained right now...

We weren't able to get this working in wolverine-frontend with babel 7. I guess this file might disappear if shiny enters wolverine's monorepo structure sometime during the coming months.

**Comment by BazanAlexandro** [comment]
Thank you!

**Comment by BazanAlexandro** [comment]
Looks like percy is not rendering images of all trysilplugs in storybook, thinking because of progressive image. Looks fine locally

**Pull request merged**